mxv/templates/mxv/index.html

```diff
@@ -8,6 +8,26 @@
 {% endblock %}
 
 {% block content %}
+    <div class="row justify-content-center">
+        <div class="col-sm-12 col-lg-6">
+            <div class="card" id="membership-card">
+                <div class="container-full p-3">
+                    <div class="row">
+                        <div class="col-8">
+                            <h2>{{ user.name }}</h2>
+                            <p>{{user.address}}</p>
+                            <p><span>e:</span> {{user.email}}</p>
```

Comment:
shouldn't this say "email" and "phone"? or if space is premium just remove it

Reply:
not per design, so am gonna leave this and address in next branch